common/ferrostar/src/navigation_session/caching/mod.rs

```diff
@@ -1,8 +1,10 @@
+#[cfg(feature = "uniffi")]
 use std::sync::{Arc, Mutex};
 
+#[cfg(feature = "uniffi")]
 use chrono::{Duration, Utc};
 
-#[cfg(all(feature = "uniffi", not(feature = "wasm_js")))]
+#[cfg(feature = "uniffi")]
```

Comment:
Bunch of minor cleanup here. Just the uniffi gate is enough. We can't target wasm with uniffi (yet?), so I changed this to match the rest of the codebase.